packages/block-library/src/navigation/view.js

```diff
@@ -39,18 +37,19 @@ const closeMenu = ( { context, selectors }, menuClosedOn ) => {
 		}
 		context.core.navigation.modal = null;
 		context.core.navigation.previousFocus = null;
-		if ( context.core.navigation.overlay ) {
+		if ( context.core.navigation.type === 'overlay' ) {
 			document.documentElement.classList.remove( 'has-modal-open' );
 		}
 	}
 };
 
-store( {
+wpStore( {
```

Comment:
I renamed this to `wpStore` to be able to use `store` instead of `args` in the functions.